docs/plugins/index.md

```diff
@@ -272,13 +272,29 @@ const char*    getSlotBytes(WrenVM* vm, int slot, int* length);
       void     removeMapValue(WrenVM* vm, int mapSlot, int keySlot, int removedValueSlot);
 ```
 
+### Embed
+
+If your plugin requires using _Wren_ files, you can _embed_ them using the built in command `--embed`.
```

Comment:
"If your plugin registers a Wren module, you can embed the source of that module in your plugin by using DOME's built-in `--embed` command, which will convert it into a C include file."